isaaclab_arena/tasks/open_door_task.py

```diff
@@ -199,11 +200,12 @@ def __post_init__(self):
                 apply_noise_during_interpolation=False,
             )
         )
-        if self.embodiment_name == "franka":
+        if self.arm_mode == "single_arm":
             self.subtask_configs["robot"] = subtask_configs
         # We need to add the left and right subtasks for GR1.
-        elif self.embodiment_name == "gr1_pink":
-            self.subtask_configs["right"] = subtask_configs
+        elif self.arm_mode in ["left", "right"]:
+            self.another_arm_mode = "left" if self.arm_mode == "right" else "right"
+            self.subtask_configs[self.arm_mode] = subtask_configs
```

Comment:
Suggestion to use the `Enum` for comparisons.

`if self.arm_mode == ArmMode.SINGLE_ARM:`